src/js/api/auth/logout.test.js

```diff
@@ -0,0 +1,17 @@
+import { logout } from './logout';
+import { index } from '../../storage/index.test';
+
+const storage = index;
+
+const token = storage;
+const profile = storage;
+
+const data = { token, profile };
+
+describe('Logout', () => {
+  it('Success', async () => {
+    global.fetch = jest.fn(() => fetchSuccess());
+    const item = logout(data);
+    expect(item).toBe(null || undefined);
```

Comment:
You need to mock localstorage and check that logout deletes the token correctly.

Reply:
```
import { logout } from './logout';
import * as storage from '../../storage/index.js';

// Mock localstorage
class LocalStorageMock {
  constructor() {
    this.value = {};
  }

  clear() {
    this.value = {};
  }

  getItem(token) {
    return this.value[token] || null;
  }

  setItem(token, value) {
    this.value[token] = String(value);
  }

  removeItem(token) {
    delete this.value[token];
  }
}

global.localStorage = new LocalStorageMock();

describe('Logout the user', () => {  
  it('Logout is a success', () => {
    logout()
    localStorage.clear(storage)
    console.log(localStorage)
  });
});
```

![image](https://user-images.githubusercontent.com/46559129/201094231-f6707955-ad0c-4004-9602-54bd22a8afa1.png)

I believe this is the expected result ?

Reply:
`logout()` is in charge of clearing localstorage. So you just need to log out then expect `localStorage.get("token")` to be null